brainstormforce/ultimate-addons-for-gutenberg
# UAG-438 Update device-specific units for control.

### Description

- https://ua-gutenberg.atlassian.net/browse/UAG-438

### Types of changes




- Improvements

### How has this been tested?

- Drag and drop block.
- Check each block has a responsive unit as per the device
- CSS work as per unit changes editor/frontend both

### Checklist:
